e2e/components/CreateFlows/CreateTearsheet-test.avt.e2e.js

```diff
@@ -0,0 +1,28 @@
+// cspell:words createtearsheet
+
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import { expect, test } from '@playwright/test';
+import { visitStory } from '../../test-utils/storybook';
+
+test.describe('CreateTearsheet @avt', () => {
+  test('@avt-default-state', async ({ page }) => {
+    await visitStory(page, {
+      component: 'CreateTearsheet',
+      id: 'ibm-products-patterns-create-flows-createtearsheet--multi-step-tearsheet',
+      globals: {
+        carbonTheme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations(
```

Comment:
Hi Nandan,
As Tearsheet is rendered only when `Open CreateTearsheet` button is clicked, we need to trigger button click I guess.
`await page.getByRole('button', { name: 'Open CreateTearsheet' }).click();`

Reply:
fixed